lib/src/modules/inventoryItem/InventoryItemStory.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { boolean } from '@storybook/addon-knobs';
+import { InventoryItem } from 'lib';
+import StoryItem from 'stories/styleguide/StoryItem';
+
+storiesOf('Components', module).add('InventoryItem', () => {
+  const isDragging = boolean('isDragging', false);
```

Comment:
🟢  You could use the `<Draggable />` component here to make it interactive vs just using knobs.

Reply:
Good idea!